webpack/JobWizard/steps/Schedule/__tests__/QueryType.test.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import * as patternfly from '@patternfly/react-core';
+import { testComponentSnapshotsWithFixtures } from '@theforeman/test';
+import { QueryType } from '../QueryType';
+
+jest.spyOn(patternfly, 'FormGroup');
+patternfly.FormGroup.mockImplementation(props => <div props={props} />);
+const fixtures = {
+  renders: {},
+};
+
+describe('QueryType', () => {
+  testComponentSnapshotsWithFixtures(QueryType, fixtures);
```

Comment:
Can you remove the snapshots testing?
we can add an integration test that just clicks on stuff and checks if API calls occurred and things like that.

Reply:
Removed, will add integration testing to the next pr as in this one is only a visual pr https://github.com/theforeman/foreman_remote_execution/pull/567